access-management/manage-private-channels.md

```diff
@@ -0,0 +1,119 @@
+---
+description: This article provides information on managing who has access or is an owner of private channels in Microsoft Teams.
+---
+
+# Manage Private Channels Access
+
+This article will help set up and manage who the owners and members are for private channels in Microsoft Teams. 
+
+There are several ways to manage and change the access to private channels from within SysKit Point.
+
```

Comment:
dodati please note (ne mora točno tako, ali važna je informacija):
The following users can manage private channel access:
* Global administrator
* Private channel owners